src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+
 public class WorkWithFile {
 
-    public void getStatistic(String fromFileName, String toFileName) {
+    public static void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
+        int buySum = 0;
+        int supplySum = 0;
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(file));
```

Comment:
we should use try with resources for closable interfaces

Reply:
needed waterfall of try with resources blocks... done!